server/src/db/migrations/20230209221711_modifyReviews.cjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * @typedef {import("knex")} Knex
+ */
+
+/**
+ * @param {Knex} knex
+ */
+exports.up = async (knex) => {
+    return knex.schema.table("reviews", (table) => {
+        table.dropColumn("comment")
```

Comment:
changing the data type of a column should be one alter migration, not dropping the column. something like:
```js
exports.up = async (knex) => {
  return knex.schema.alterTable("reviews", (table) => {
    table.text("comment").notNullable().alter();
  });
};

/**
 * @param {Knex} knex
 */
exports.down = (knex) => {
  return knex.schema.alterTable("reviews", (table) => {
    table.string("comment").notNullable().alter();
  });
};

```
where `up` action indicates how to change the column going forward with migrations, and the `down` resets the column back to a string when rolled back